README.md

```diff
@@ -24,3 +24,34 @@ translators in seconds.
 Zanata is Free software, licensed under the [LGPL][].
 
 [LGPL]: http://www.gnu.org/licenses/lgpl-2.1.html
+
+Developers
+----------
+
+*Prerequisites*
```

Comment:
You can use different levels of heading for these sections, rather than just make them bold/italic:

```md
# h1
## h2
### h3
#### etc
```